### DISCUSSION TITLE
Warehouse (2023) High Rounds And Records

### DISCUSSION THREAD

**Discussion by bignoobus** [nzp-team/nzportable#506] [Records and Speedruns]
![fte-20231202135832-0](https://github.com/nzp-team/nzportable/assets/105007743/fa48192f-d452-475e-94c9-8276a47b8539)

well its not a 50 thats for sure

**Comment by MisaSM** [comment]
![almost50](https://github.com/nzp-team/nzportable/assets/101074089/da271331-3a22-4d7b-b2de-8f267aa0af00)

Slightly fumbled it at the end

**Comment by DGxShAdOwZz** [comment]
![WR1](https://github.com/nzp-team/nzportable/assets/128944577/38aac6f6-2d9e-4332-b7ae-4fdf5c053540)

Lost concentration lmao

**Comment by Vinxent9** [comment]
![wh23](https://github.com/nzp-team/nzportable/assets/56005655/62a3111a-2234-4f9f-966a-dc423ecfcd86)

will try again next time

**Comment by MotoLegacy** [comment]
Very nice!
